### DISCUSSION TITLE
Don't group directories at the top?

### DISCUSSION THREAD

**Discussion by tim-tx** [jarun/nnn#1791] [Q&A] [answered]
How can I make nnn mix files and directories when sorting? Default behavior is that directories are grouped at the top, files at the bottom. I did not find anything in the man page / program options. I thought this had something to do with LC_COLLATE but I found with `ls` you have to opt in to this grouping behavior with `ls --group-directories-first`.

**Comment by N-R-K** [comment, marked as answer]
> How can I make nnn mix files and directories when sorting?

That behaviour is not configurable AFAIK. So you'd need to modify the source code if you want to disable it.
